packages/slice-machine/server/src/api/tracking.ts

```diff
@@ -37,14 +46,23 @@ export function sendEvents(
       });
     }
   } else if (isTrackingEvent(event)) {
+    const maybeNodeVersion =
+      event.name === EventNames.PageView ? { nodeVersion } : {};
+
     analytics.track({
       event: event.name,
       properties: {
         ...event.props,
-        nodeVersion: process.versions.node,
+        ...maybeNodeVersion,
       },
       ...(userId !== undefined ? { userId } : { anonymousId }),
-      context: { groupId: { Repository: repositoryName } },
+      context: {
+        app: {
+          name,
```

Comment:
would it be possible to add this to group and identify calls as well so the APP is registered on everything we send to Segment. 🙏